src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -496,57 +507,93 @@ void getIndividualInterpreterSignLanguageTest(String signLang, String expected)
-        String result = getIndividualPreferredHearingChannel(null, null).orElse(null);
+        HearingOptions hearingOptions = HearingOptions.builder().build();
+        String result = getIndividualPreferredHearingChannel(null, null, hearingOptions);
         assertThat(result).isNull();
     }
 
-    @DisplayName("When hearing type oral and video then return LOV not attending")
+    @DisplayName("When hearing type oral and video then return LOV VIDEO")
     @Test
     void getIndividualPreferredHearingChannelOralVideoTest() {
-        HearingSubtype hearingSubtype = HearingSubtype.builder().wantsHearingTypeVideo("Yes").build();
-        String result = getIndividualPreferredHearingChannel("oral", hearingSubtype).orElse(null);
+        HearingSubtype hearingSubtype = HearingSubtype.builder().wantsHearingTypeVideo("Yes").hearingVideoEmail("test@email.com").build();
+        HearingOptions hearingOptions = HearingOptions.builder().wantsToAttend("yes").build();
+        String result = getIndividualPreferredHearingChannel("oral", hearingSubtype, hearingOptions);
         assertThat(result).isEqualTo(HearingChannel.VIDEO.getHmcReference());
     }
 
-    @DisplayName("When hearing type oral and telephone then return LOV not attending")
+    @DisplayName("When hearing type oral and telephone then return LOV TELEPHONE")
     @Test
     void getIndividualPreferredHearingChannelOralTelephoneTest() {
-        HearingSubtype hearingSubtype = HearingSubtype.builder().wantsHearingTypeTelephone("Yes").build();
-        String result = getIndividualPreferredHearingChannel("oral", hearingSubtype).orElse(null);
+        HearingSubtype hearingSubtype = HearingSubtype.builder().wantsHearingTypeTelephone("Yes").hearingTelephoneNumber("01111234567").build();
+        HearingOptions hearingOptions = HearingOptions.builder().wantsToAttend("yes").build();
+        String result = getIndividualPreferredHearingChannel("oral", hearingSubtype, hearingOptions);
         assertThat(result).isEqualTo(HearingChannel.TELEPHONE.getHmcReference());
     }
 
-    @DisplayName("When hearing type oral and face to face then return LOV not attending")
+    @DisplayName("When wantsToAttend is yes, and wantsHearingType telephone but hearingTelephoneNumber is not set throw IllegalStateException")
+    @Test
+    void getIndividualPreferredHearingChannelNullWhenMissingPartialRequirementsTelephoneExample() {
+        HearingSubtype hearingSubtype = HearingSubtype.builder().wantsHearingTypeTelephone("Yes").build();
+        HearingOptions hearingOptions = HearingOptions.builder().wantsToAttend("yes").build();
+
+        IllegalStateException thrown = assertThrows(
+            IllegalStateException.class,
+            () -> getIndividualPreferredHearingChannel("oral", hearingSubtype, hearingOptions),
+            "Expected doThing() to throw, but it didn't"
+        );
+
+        assertTrue(thrown.getMessage().contains("Failed to determine a preferred hearing channel"));
```

Comment:
(Minor) can use assertJ's assertThatExceptionOfType(IllegalStateException.class).isThrownBy(()-> getIndividualPreferredHearingChannel) to do the assertion in on eline

Reply:
Done